README.md

```diff
@@ -0,0 +1,33 @@
+<div align="center">
+  <h1>Untitled Design System</h1>
+
+</div>
+
+## Getting Started
+
+Before starting make sure [node](https://nodejs.org/en/) and [pnpm](https://pnpm.io) is installed.
+
+### Install
+
+After cloning the source code, run `pnpm` to install dependencies.
+
+```bash
+git clone https://github.com/workleap/wl-untitled.git
+cd wl-untitled
+pnpm i
+```
+
+### Scripts
+
+- `pnpm doc:start`: run documentation application
+- `pnpm doc:storybook`: run Storybook
+- `pnpm test`: run tests
+- `pnpm lint`: check if the JS, TS and style correspond on standard
+
+## 🤝 Contributing
+
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change. If you're interested, definitely check out our Contributing Guide!
```

Comment:
Should not this be a discussion instead of an issue? We tend to keep issues for issues, and discussion for discussing major changes.

Reply:
Yeah we now expect to use [Github Discussions](https://docs.github.com/en/discussions) for those rather than issues. It's more adapted and offer a great separation between actual bugs and dicussions / questions.